openfecwebapp/templates/layouts/main.html

```diff
@@ -16,6 +16,7 @@
   {% endif %}
   {% if api_location %}
   <script>
+    ANALYTICS = {{ use_analytics|json }};
```

Comment:
As @adborden found, it's our own custom filter: https://github.com/18F/openFEC-web-app/blob/3df7e7fad9ed1d2413a13270e026f619c7bdda40/openfecwebapp/filters.py#L33